.github/workflows/build.yml

```diff
@@ -22,6 +22,8 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: install
       run: npm ci
+    - name: test
+      run: npm t
```

Comment:
`npm t` instead of `npm test`, didn't know you could do this?

Reply:
I'm tempted to rename the `test` folder to `tests` to keep it in line with Alpine 🤔

Reply:
oh wait it's already `tests` 😂